NEWS.md

```diff
@@ -22,6 +22,8 @@
 
 1. `unique()` returns a copy the case when `nrows(x) <= 1` instead of a mutable alias, [#5932](https://github.com/Rdatatable/data.table/pull/5932). This is consistent with existing `unique()` behavior when the input has no duplicates but more than one row. Thanks to @brookslogan for the report and @dshemetov for the fix.
 
+2. `fwrite(..., row.names=TRUE)` with `matrix` always printed row numbers as `row.names` even if `matrix` had a `row.names` attribute, [#5315](https://github.com/Rdatatable/data.table/issues/5315). Thanks to @Liripo for the report, and @ben-schwen for the fix.
```

Comment:
It's not clear just reading the NEWS entry what `matrix` means here. Does it mean `fwrite(x, row.names=TRUE)`, where `is.matrix(x)`? Maybe this would be clearer:

```suggestion
2. `fwrite(x, ..., row.names=TRUE)` with `x` a `matrix` writes `row.names` when present, not row numbers, [#5315](https://github.com/Rdatatable/data.table/issues/5315). Thanks to @Liripo for the report, and @ben-schwen for the fix.
```